src/scr_conf.h

```diff
@@ -152,6 +152,11 @@
 #define SCR_FILE_BUF_SIZE (1024*1024)
 #endif
 
+/* whether file metadata should also be copied */
+#ifndef SCR_COPY_METADATA
+#define SCR_COPY_METADATA (0)
```

Comment:
Let's default this to 1

Reply:
This defaults (0) mimics that one currently used in AXL: https://github.com/ECP-VeloC/AXL/blob/a799cd9b7a406a80173ecb2bf3a32000f432f8ce/src/axl.c#L258

There is an instance of meta data copying in SCR, however that is in the standalone `scr_copy` tool which does indeed default to copying metadata.

Is the preference to update defaults in both libSCR and libAXL to match `scr_copy` (namely 1) or would there be more good in not changing the default that AXL will see when called via SCR (SCR's default will override AXL's).

Reply:
Right, good reasoning to default this the same as AXL as a starting point.  As a default in SCR, we'll want to copy the metadata (unless the user tells us not to).  I'm unsure whether we should make that the default for AXL.  Maybe there are AXL users who don't want to bother.  I feel like we might want to swap it there, too.  In any case, we can have SCR configure AXL to copy that metadata if we opt to not copy metadata by default in AXL.

BTW, I can imagine at some point, we may need to split this so the user can specify more precisely what parts of the metadata (uid, gid, mode bits, timestamps) are copied.  I think some people may prefer to copy uid/gid/mode but let the timestamps be updated.  That way, the timestamps would represent the time the files actually showed up on the parallel file system rather than the time they were created in SCR cache.  Though, that's a detail we can worry about later.  For now a single configuration param to copy all metadata will do.

Reply:
I changed the default to 1 in 9e4ba7bbaea96d691bb9caf96b03b5afb1c2528f.

Reply:
I added rst docs for `SCR_Config` (and `SCR_Configf`) in 231e510.